docs/advanced.rst

```diff
@@ -41,10 +41,57 @@ that need it, like the ``ItemWebPage``.
 You are able to implement your own providers in order to extend or override
 current scrapy-poet behavior.
 
+Currently, scrapy-poet is able to inject instances of the following
+classes as *provider* dependencies:
+
+- :class:`~scrapy.http.Request`
+- :class:`~scrapy.http.Response`
+- :class:`~scrapy.crawler.Crawler`
+- :class:`~scrapy.settings.Settings`
+- :class:`~scrapy.statscollectors.StatsCollector`
+
 .. warning::
 
-    Currently, scrapy-poet is only able to inject ``Response`` and
-    :class:`~.DummyResponse` instances as *provider* dependencies.
+    If your provider needs to use a subclass of Response,
```

Comment:
I'm not sure this explanation is now that relevant in that form. The developer asks for a `Response` in the provider and can receive a subclass of it, but this is also the case in the regular Scrapy callbacks. So nothing new here, right?.

Reply:
Yes, that's correct. I've tried to simplify the docs mentioning something like this.

Reply:
Could you check it, please?